components/ILIAS/StudyProgramme/PRIVACY.md

```diff
@@ -0,0 +1,159 @@
+# Individual Assessment Privacy
+
+Disclaimer: This documentation does not warrant completeness or correctness. Please
+report any missing or wrong information using the [ILIAS issue tracker](https://mantis.ilias.de)
+or contribute a fix via [Pull Request](docs/development/contributing.md#pull-request-to-the-repositories).
+
+### General Information
+
+The Study Programme combines Courses and other Study Programmes to a curriculum. Automations can add Users and Courses based on certain triggers.
+When any one Course within a Study Programme is passed, the Study Program awards the points needed for its completion. Alternatively, a Study Programmes takes the sum of points from all Study Programmes it contains towards the required point score for completion. This leads to a qualification which may create a Certificate including a PDF for printing.
```

Comment:
Please change Study Program to **Study Programme**.

Reply:
changed